### PR TITLE
[SELS-Feature][FE] Setup Basic Routes

### PR DESCRIPTION
**Pull request by IanUrriza10**
## [Asana Link]
https://app.asana.com/0/1203212597546540/1203223794310940
## [Expected Output]
Accessing localhost:3000/#/ should return a homepage with header and footer.
Accessing any other url should return a 404 not found page with header and footer.

## [Screenshots]
image
image

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,25 +1,38 @@
...
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="App ui component">
+      <HashRouter>
+        <Header />
+        <Routes>
+          <Route
```
Comment (roseaugusto) [inline frontend/src/App.js:15]: create a file `routes.js` and import here instead of directly putting in `app.js`

Comment (IanUrriza10) [reply frontend/src/App.js:15]: Should i put it on the same level as App.js or put it on a specific folder?

File: frontend/src/routes/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import { HashRouter, Routes, Route } from "react-router-dom";
+import Footer from "../components/Footer";
+
+import Header from "../components/Header";
+import Homepage from "../pages/Homepage";
+import Login from "../pages/Login";
+import Register from "../pages/Register";
+import PageNotFound from "../pages/PageNotFound";
+
+const Directory = () => {
```
Comment (roseaugusto) [inline frontend/src/routes/index.js:10]: since this is an `index.js` file, please put same name as `Routes` instead of directory

Comment (IanUrriza10) [reply frontend/src/routes/index.js:10]: it would be hard since Routes is already a used name in react-router

**Review by roseaugusto — APPROVED** [review]

**Pull request merged**